lib/sdk-test-helpers.ts

```diff
@@ -14,7 +14,8 @@
  * limitations under the License.
  */
 
-import expect from 'expect';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { expect } from '@jest/globals';
```

Comment:
This import is changed from "expect" to "jest", since that is what will be available in the SDK projects by default.